pinot-common/src/main/java/org/apache/pinot/common/utils/request/RequestUtils.java

```diff
@@ -127,62 +126,41 @@ public static Expression createNewLiteralExpression() {
     return expression;
   }
 
-  public static Expression getLiteralExpression(String value) {
```

Comment:
Why remove this?

Will it default to `getLiteralExpression (Object )` ?

Reply:
It is still there, but somehow `git` does not line them up.